ports/nrf/mphalport.c

```diff
@@ -70,7 +177,11 @@ int mp_hal_stdin_rx_chr(void) {
         if (MP_STATE_PORT(board_stdio_uart) != NULL && uart_rx_any(MP_STATE_PORT(board_stdio_uart))) {
             return uart_rx_char(MP_STATE_PORT(board_stdio_uart));
         }
+        #if MICROPY_PY_TIME_TICKS
+        MICROPY_EVENT_POLL_HOOK
+        #else
```

Comment:
could it just use `MICROPY_EVENT_POLL_HOOK` unconditionally (ie in both cases here)?

Reply:
Yes I was thinking about that too.. (since there was already a WFI statement). But I wanted it to behave like the old, legacy solution if `MICROPY_PY_TIME_TICKS` was disabled. But I can easily enable it always, no problem.

Reply:
aah, since `MICROPY_ENABLE_SCHEDULER` is disabled on nrf it won't make a difference, `mp_handle_pending()` won't do anything except check for pending exceptions, but at the REPL (where this function is used) ctrl-C (the only possible pending exception) is disabled

Reply:
ok